tests/lib/bluetooth/ble_conn_params/CMakeLists.txt

```
@@ -0,0 +1,73 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+
+project(unit_test_ble_conn_params)
+
+set(SOFTDEVICE_VARIANT "s115")
+set(SOFTDEVICE_INCLUDE_DIR "${ZEPHYR_NRF_BM_MODULE_DIR}/components/softdevice/\
+${SOFTDEVICE_VARIANT}/${SOFTDEVICE_VARIANT}_API/include")
+
+cmock_handle(${SOFTDEVICE_INCLUDE_DIR}/ble.h)
+cmock_handle(${SOFTDEVICE_INCLUDE_DIR}/ble_gatts.h
+  WORD_EXCLUDE
+  "__STATIC_INLINE")
+cmock_handle(${SOFTDEVICE_INCLUDE_DIR}/ble_gattc.h
+  WORD_EXCLUDE
+  "__STATIC_INLINE")
```

Word exclude of static inline is not needed when defining `SUPPRESS_INLINE_IMPLEMENTATION`-
```suggestion
cmock_handle(${SOFTDEVICE_INCLUDE_DIR}/ble_gatts.h)
cmock_handle(${SOFTDEVICE_INCLUDE_DIR}/ble_gattc.h)
```